### ISSUE TITLE
"Cannot compute the outputs" using SSD Mobilenet v1 coco

### ISSUE THREAD

**Issue by blechatellier** [tensorflow/tfjs#1793]
I've successfully converted [ssd mobilenet v1 coco](http://download.tensorflow.org/models/object_detection/ssd_mobilenet_v1_coco_2018_01_28.tar.gz) from the official [tensorflow object detection repo](https://github.com/tensorflow/models/blob/master/research/object_detection/g3doc/detection_model_zoo.md) using tfjsconverter but running into issues predicting.

### Python dependencies
tensorflow 1.14.0
tensorflowjs 1.2.6

### tfjsconverter code
```
tensorflowjs_converter \
  --input_format=tf_saved_model \
  --output_format=tfjs_graph_model \
  --signature_name=serving_default \
  --saved_model_tags=serve \
  ssd_mobilenet_v1_coco_2018_01_28/saved_model \
  ssd_mobilenet_v1_coco_2018_01_28/web_model
```

### JS dependencies
@tensorflow/tfjs-node 1.2.6
canvas 2.5.0

### JS code
```
import * as tf from "@tensorflow/tfjs-node";
import { createCanvas, Image } from "canvas";

(async () => {
  const MODEL_URL = `file://./ssd_mobilenet_v1_coco_2018_01_28/web_model/model.json`;
  const model = await tf.loadGraphModel(MODEL_URL);
  const img = new Image();
  img.src = "./images/faces.jpg";
  const canvas = createCanvas(img.width, img.height);
  const ctx = canvas.getContext("2d");
  ctx.drawImage(img, 0, 0);
  const image_tensor = tf.expandDims(tf.browser.fromPixels(canvas as any));
  const predict = await model.executeAsync({ image_tensor });
})();
```

### Error
```
(node:292) UnhandledPromiseRejectionWarning: Error: Cannot compute the outputs [unused_control_flow_input_99,unused_control_flow_input_98,unused_control_flow_input_97,unused_control_flow_input_96,unused_control_flow_input_95,unused_control_flow_input_94,unused_control_flow_input_93,unused_control_flow_input_92,unused_control_flow_input_91,unused_control_flow_input_90,unused_control_flow_input_9,unused_control_flow_input_89,unused_control_flow_input_88,unused_control_flow_input_87,unused_control_flow_input_86,unused_control_flow_input_85,unused_control_flow_input_84,unused_control_flow_input_83,unused_control_flow_input_82,unused_control_flow_input_81,unused_control_flow_input_80,unused_control_flow_input_8,unused_control_flow_input_79,unused_control_flow_input_78,unused_control_flow_input_77,unused_control_flow_input_76,unused_control_flow_input_75,unused_control_flow_input_74,unused_control_flow_input_73,unused_control_flow_input_72,unused_control_flow_input_71,unused_control_flow_input_70,unused_control_flow_input_7,unused_control_flow_input_69,unused_control_flow_input_68,unused_control_flow_input_67,unused_control_flow_input_66,unused_control_flow_input_65,unused_control_flow_input_64,unused_control_flow_input_63,unused_control_flow_input_62,unused_control_flow_input_61,unused_control_flow_input_60,unused_control_flow_input_6,unused_control_flow_input_59,unused_control_flow_input_58,unused_control_flow_input_57,unused_control_flow_input_56,unused_control_flow_input_55,unused_control_flow_input_54,unused_control_flow_input_53,unused_control_flow_input_52,unused_control_flow_input_51,unused_control_flow_input_50,unused_control_flow_input_5,unused_control_flow_input_49,unused_control_flow_input_48,unused_control_flow_input_47,unused_control_flow_input_46,unused_control_flow_input_45,unused_control_flow_input_44,unused_control_flow_input_43,unused_control_flow_input_42,unused_control_flow_input_41,unused_control_flow_input_40,unused_control_flow_input_4,unused_control_flow_input_39,unused_control_flow_input_38,unused_control_flow_input_37,unused_control_flow_input_36,unused_control_flow_input_35,unused_control_flow_input_34,unused_control_flow_input_33,unused_control_flow_input_32,unused_control_flow_input_31,unused_control_flow_input_30,unused_control_flow_input_3,unused_control_flow_input_29,unused_control_flow_input_28,unused_control_flow_input_27,unused_control_flow_input_26,unused_control_flow_input_25,unused_control_flow_input_24,unused_control_flow_input_237,unused_control_flow_input_236,unused_control_flow_input_235,unused_control_flow_input_234,unused_control_flow_input_233,unused_control_flow_input_232,unused_control_flow_input_231,unused_control_flow_input_230,unused_control_flow_input_23,unused_control_flow_input_229,unused_control_flow_input_228,unused_control_flow_input_227,unused_control_flow_input_226,unused_control_flow_input_225,unused_control_flow_input_224,unused_control_flow_input_223,unused_control_flow_input_222,unused_control_flow_input_221,unused_control_flow_input_220,unused_control_flow_input_22,unused_control_flow_input_219,unused_control_flow_input_218,unused_control_flow_input_217,unused_control_flow_input_216,unused_control_flow_input_215,unused_control_flow_input_214,unused_control_flow_input_213,unused_control_flow_input_212,unused_control_flow_input_211,unused_control_flow_input_210,unused_control_flow_input_21,unused_control_flow_input_209,unused_control_flow_input_208,unused_control_flow_input_207,unused_control_flow_input_206,unused_control_flow_input_205,unused_control_flow_input_204,unused_control_flow_input_203,unused_control_flow_input_202,unused_control_flow_input_201,unused_control_flow_input_200,unused_control_flow_input_20,unused_control_flow_input_2,unused_control_flow_input_199,unused_control_flow_input_198,unused_control_flow_input_197,unused_control_flow_input_196,unused_control_flow_input_195,unused_control_flow_input_194,unused_control_flow_input_193,unused_control_flow_input_192,unused_control_flow_input_191,unused_control_flow_input_190,unused_control_flow_input_19,unused_control_flow_input_189,unused_control_flow_input_188,unused_control_flow_input_187,unused_control_flow_input_186,unused_control_flow_input_185,unused_control_flow_input_184,unused_control_flow_input_183,unused_control_flow_input_182,unused_control_flow_input_181,unused_control_flow_input_180,unused_control_flow_input_18,unused_control_flow_input_179,unused_control_flow_input_178,unused_control_flow_input_177,unused_control_flow_input_176,unused_control_flow_input_175,unused_control_flow_input_174,unused_control_flow_input_173,unused_control_flow_input_172,unused_control_flow_input_171,unused_control_flow_input_170,unused_control_flow_input_17,unused_control_flow_input_169,unused_control_flow_input_168,unused_control_flow_input_167,unused_control_flow_input_166,unused_control_flow_input_165,unused_control_flow_input_164,unused_control_flow_input_163,unused_control_flow_input_162,unused_control_flow_input_161,unused_control_flow_input_160,unused_control_flow_input_16,unused_control_flow_input_159,unused_control_flow_input_158,unused_control_flow_input_157,unused_control_flow_input_156,unused_control_flow_input_155,unused_control_flow_input_154,unused_control_flow_input_153,unused_control_flow_input_152,unused_control_flow_input_151,unused_control_flow_input_150,unused_control_flow_input_15,unused_control_flow_input_149,unused_control_flow_input_148,unused_control_flow_input_147,unused_control_flow_input_146,unused_control_flow_input_145,unused_control_flow_input_144,unused_control_flow_input_143,unused_control_flow_input_142,unused_control_flow_input_141,unused_control_flow_input_140,unused_control_flow_input_14,unused_control_flow_input_139,unused_control_flow_input_138,unused_control_flow_input_137,unused_control_flow_input_136,unused_control_flow_input_135,unused_control_flow_input_134,unused_control_flow_input_133,unused_control_flow_input_132,unused_control_flow_input_131,unused_control_flow_input_130,unused_control_flow_input_13,unused_control_flow_input_129,unused_control_flow_input_128,unused_control_flow_input_127,unused_control_flow_input_126,unused_control_flow_input_125,unused_control_flow_input_124,unused_control_flow_input_123,unused_control_flow_input_122,unused_control_flow_input_121,unused_control_flow_input_120,unused_control_flow_input_12,unused_control_flow_input_119,unused_control_flow_input_118,unused_control_flow_input_117,unused_control_flow_input_116,unused_control_flow_input_115,unused_control_flow_input_114,unused_control_flow_input_113,unused_control_flow_input_112,unused_control_flow_input_111,unused_control_flow_input_110,unused_control_flow_input_11,unused_control_flow_input_109,unused_control_flow_input_108,unused_control_flow_input_107,unused_control_flow_input_106,unused_control_flow_input_105,unused_control_flow_input_104,unused_control_flow_input_103,unused_control_flow_input_102,unused_control_flow_input_101,unused_control_flow_input_100,unused_control_flow_input_10,unused_control_flow_input_1,unused_control_flow_input] from the provided inputs [image_tensor]. Consider providing the following inputs: [unused_control_flow_input,unused_control_flow_input_1,unused_control_flow_input_10,unused_control_flow_input_100,unused_control_flow_input_101,unused_control_flow_input_102,unused_control_flow_input_103,unused_control_flow_input_104,unused_control_flow_input_105,unused_control_flow_input_106,unused_control_flow_input_107,unused_control_flow_input_108,unused_control_flow_input_109,unused_control_flow_input_11,unused_control_flow_input_110,unused_control_flow_input_111,unused_control_flow_input_112,unused_control_flow_input_113,unused_control_flow_input_114,unused_control_flow_input_115,unused_control_flow_input_116,unused_control_flow_input_117,unused_control_flow_input_118,unused_control_flow_input_119,unused_control_flow_input_12,unused_control_flow_input_120,unused_control_flow_input_121,unused_control_flow_input_122,unused_control_flow_input_123,unused_control_flow_input_124,unused_control_flow_input_125,unused_control_flow_input_126,unused_control_flow_input_127,unused_control_flow_input_128,unused_control_flow_input_129,unused_control_flow_input_13,unused_control_flow_input_130,unused_control_flow_input_131,unused_control_flow_input_132,unused_control_flow_input_133,unused_control_flow_input_134,unused_control_flow_input_135,unused_control_flow_input_136,unused_control_flow_input_137,unused_control_flow_input_138,unused_control_flow_input_139,unused_control_flow_input_14,unused_control_flow_input_140,unused_control_flow_input_141,unused_control_flow_input_142,unused_control_flow_input_143,unused_control_flow_input_144,unused_control_flow_input_145,unused_control_flow_input_146,unused_control_flow_input_147,unused_control_flow_input_148,unused_control_flow_input_149,unused_control_flow_input_15,unused_control_flow_input_150,unused_control_flow_input_151,unused_control_flow_input_152,unused_control_flow_input_153,unused_control_flow_input_154,unused_control_flow_input_155,unused_control_flow_input_156,unused_control_flow_input_157,unused_control_flow_input_158,unused_control_flow_input_159,unused_control_flow_input_16,unused_control_flow_input_160,unused_control_flow_input_161,unused_control_flow_input_162,unused_control_flow_input_163,unused_control_flow_input_164,unused_control_flow_input_165,unused_control_flow_input_166,unused_control_flow_input_167,unused_control_flow_input_168,unused_control_flow_input_169,unused_control_flow_input_17,unused_control_flow_input_170,unused_control_flow_input_171,unused_control_flow_input_172,unused_control_flow_input_173,unused_control_flow_input_174,unused_control_flow_input_175,unused_control_flow_input_176,unused_control_flow_input_177,unused_control_flow_input_178,unused_control_flow_input_179,unused_control_flow_input_18,unused_control_flow_input_180,unused_control_flow_input_181,unused_control_flow_input_182,unused_control_flow_input_183,unused_control_flow_input_184,unused_control_flow_input_185,unused_control_flow_input_186,unused_control_flow_input_187,unused_control_flow_input_188,unused_control_flow_input_189,unused_control_flow_input_19,unused_control_flow_input_190,unused_control_flow_input_191,unused_control_flow_input_192,unused_control_flow_input_193,unused_control_flow_input_194,unused_control_flow_input_195,unused_control_flow_input_196,unused_control_flow_input_197,unused_control_flow_input_198,unused_control_flow_input_199,unused_control_flow_input_2,unused_control_flow_input_20,unused_control_flow_input_200,unused_control_flow_input_201,unused_control_flow_input_202,unused_control_flow_input_203,unused_control_flow_input_204,unused_control_flow_input_205,unused_control_flow_input_206,unused_control_flow_input_207,unused_control_flow_input_208,unused_control_flow_input_209,unused_control_flow_input_21,unused_control_flow_input_210,unused_control_flow_input_211,unused_control_flow_input_212,unused_control_flow_input_213,unused_control_flow_input_214,unused_control_flow_input_215,unused_control_flow_input_216,unused_control_flow_input_217,unused_control_flow_input_218,unused_control_flow_input_219,unused_control_flow_input_22,unused_control_flow_input_220,unused_control_flow_input_221,unused_control_flow_input_222,unused_control_flow_input_223,unused_control_flow_input_224,unused_control_flow_input_225,unused_control_flow_input_226,unused_control_flow_input_227,unused_control_flow_input_228,unused_control_flow_input_229,unused_control_flow_input_23,unused_control_flow_input_230,unused_control_flow_input_231,unused_control_flow_input_232,unused_control_flow_input_233,unused_control_flow_input_234,unused_control_flow_input_235,unused_control_flow_input_236,unused_control_flow_input_237,unused_control_flow_input_24,unused_control_flow_input_25,unused_control_flow_input_26,unused_control_flow_input_27,unused_control_flow_input_28,unused_control_flow_input_29,unused_control_flow_input_3,unused_control_flow_input_30,unused_control_flow_input_31,unused_control_flow_input_32,unused_control_flow_input_33,unused_control_flow_input_34,unused_control_flow_input_35,unused_control_flow_input_36,unused_control_flow_input_37,unused_control_flow_input_38,unused_control_flow_input_39,unused_control_flow_input_4,unused_control_flow_input_40,unused_control_flow_input_41,unused_control_flow_input_42,unused_control_flow_input_43,unused_control_flow_input_44,unused_control_flow_input_45,unused_control_flow_input_46,unused_control_flow_input_47,unused_control_flow_input_48,unused_control_flow_input_49,unused_control_flow_input_5,unused_control_flow_input_50,unused_control_flow_input_51,unused_control_flow_input_52,unused_control_flow_input_53,unused_control_flow_input_54,unused_control_flow_input_55,unused_control_flow_input_56,unused_control_flow_input_57,unused_control_flow_input_58,unused_control_flow_input_59,unused_control_flow_input_6,unused_control_flow_input_60,unused_control_flow_input_61,unused_control_flow_input_62,unused_control_flow_input_63,unused_control_flow_input_64,unused_control_flow_input_65,unused_control_flow_input_66,unused_control_flow_input_67,unused_control_flow_input_68,unused_control_flow_input_69,unused_control_flow_input_7,unused_control_flow_input_70,unused_control_flow_input_71,unused_control_flow_input_72,unused_control_flow_input_73,unused_control_flow_input_74,unused_control_flow_input_75,unused_control_flow_input_76,unused_control_flow_input_77,unused_control_flow_input_78,unused_control_flow_input_79,unused_control_flow_input_8,unused_control_flow_input_80,unused_control_flow_input_81,unused_control_flow_input_82,unused_control_flow_input_83,unused_control_flow_input_84,unused_control_flow_input_85,unused_control_flow_input_86,unused_control_flow_input_87,unused_control_flow_input_88,unused_control_flow_input_89,unused_control_flow_input_9,unused_control_flow_input_90,unused_control_flow_input_91,unused_control_flow_input_92,unused_control_flow_input_93,unused_control_flow_input_94,unused_control_flow_input_95,unused_control_flow_input_96,unused_control_flow_input_97,unused_control_flow_input_98,unused_control_flow_input_99]. Alternatively, to avoid the dynamic ops, use model.execute() and specify the inputs [Postprocessor/BatchMultiClassNonMaxSuppression/map/TensorArrayStack_2/TensorArrayGatherV3]
```

**Comment by rthadur** [comment]
@blechatellier can you try with tfjs-node 1.2.6 ?

**Comment by blechatellier** [comment]
@rthadur just tried and running into the same issue, do I have to manually pass in the outputs I want to `executeAsync`?

**Comment by aohan237** [comment]
have the same problem.

the converter generates lots of unused_control_flow_input, which makes the input shape changed.

i think maybe it is the converter problem.

> successfully converted
@blechatellier  by the way, maybe  we just make the converter run right, does not mean, it convert the model right.

**Comment by blechatellier** [comment]
I've tried with tfjsconverter 0.8.6 and it works, returning the expected outputs.

**Comment by pyu10055** [comment]
@blechatellier sorry for the issues. With the latest converter, there are bunch unused_control_flow_input generate, which makes the JS executor confused about the default output nodes. There is a bug reported for TF 2.0. Until the bug is fixed in TF 2.0 and it is stable enough for us to use, your best option is to specify the output nodes. Thanks.

**Comment by jocca1985** [comment]
I removed manually all unused_control_flow_input records in model.json and it works.
